### DISCUSSION TITLE
Show search history with recent searches

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#695] [Q&A] [answered]
Show search history with recent searches.

- History remembers the recent free text searches

- Recent searches can be used again with one click

- History can be cleared in search advanced settings

<img src="https://github.com/user-attachments/assets/379034cc-979d-4357-9f2e-346a070a679a" width="50%"/>

**Comment by devemux86** [comment]
Available in the latest:
- https://github.com/devemux86/cruiser/discussions/690

**Comment by devemux86** [comment, marked as answer]
- Show search history with recent searches is available in https://github.com/devemux86/cruiser/discussions/701

**Comment by MotoUKRider** [comment]
Consider adding an :x: at the right-hand-side of each historic entry to easily clear unwanted "recents" ?

**Reply by devemux86** [reply]
@MotoUKRider Thanks for this nice suggestion. :+1: 

It will be available in the next update.
(and we can remove the clear setting)

<img src="https://github.com/user-attachments/assets/774a0436-3eef-45ec-9be1-1ef9f5a67715" width="50%"/>

**Reply by MotoUKRider** [reply]
@devemux86 Thanks.  The "History" is a nice feature although I found (mainly through *"let's try and break it"*) it filled-up quickly and the "clear" function was a little, ahem, buried away.
